src/main/java/de/bayern/gdi/App.java

```diff
@@ -106,7 +106,9 @@ private static void helpAndExit() {
      * @param args the command line arguments
      */
     public static void main(String[] args) {
-
+        //http://docs.geotools.org/latest/userguide/library/
+        // referencing/order.html
+        System.setProperty("org.geotools.referencing.forceXY", "true");
```

Comment:
Mhm ... It would be nice to move this code a bit nearer to the geo-related stuff.